Many modules are hidden in this stack. Use "module --show_hidden spider SOFTWARE" if you are not able to find the required software

The following have been reloaded with a version change:
  1) stack/.2024-04-silent => stack/2024-04


----------------------------------------------------------------------------
  gcc: gcc/8.5.0
----------------------------------------------------------------------------

    You will need to load all module(s) on any one of the lines below before the "gcc/8.5.0" module is available to load.

      stack/.2024-04-silent
      stack/2024-04
 
    Help:
      The GNU Compiler Collection includes front ends for C, C++, Objective-C,
      Fortran, Ada, and Go, as well as libraries for these languages.


 

Site groups:
        site  group
0    AR-SLu      6
1    AT-Neu      3
2    AU-ASM      7
3    AU-Cpr      4
4    AU-Cum      6
..      ...    ...
285  US-xTR      7
286  US-xUK      1
287  US-xUN      4
288  US-xYE      0
289  ZM-Mon      0

[290 rows x 2 columns]
/cluster/home/akmete/Setup/.venv/lib/python3.11/site-packages/joblib/externals/loky/process_executor.py:752: UserWarning: A worker stopped while some jobs were given to the executor. This can be caused by a too short worker timeout or by a memory leak.
  warnings.warn(
Traceback (most recent call last):
  File "/cluster/project/math/akmete/MSc/LOGOCVXGB.py", line 288, in <module>
    results = Parallel(n_jobs=10)(
              ^^^^^^^^^^^^^^^^^^^^
  File "/cluster/home/akmete/Setup/.venv/lib/python3.11/site-packages/joblib/parallel.py", line 2007, in __call__
    return output if self.return_generator else list(output)
                                                ^^^^^^^^^^^^
  File "/cluster/home/akmete/Setup/.venv/lib/python3.11/site-packages/joblib/parallel.py", line 1650, in _get_outputs
    yield from self._retrieve()
  File "/cluster/home/akmete/Setup/.venv/lib/python3.11/site-packages/joblib/parallel.py", line 1754, in _retrieve
    self._raise_error_fast()
  File "/cluster/home/akmete/Setup/.venv/lib/python3.11/site-packages/joblib/parallel.py", line 1789, in _raise_error_fast
    error_job.get_result(self.timeout)
  File "/cluster/home/akmete/Setup/.venv/lib/python3.11/site-packages/joblib/parallel.py", line 745, in get_result
    return self._return_or_raise()
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/cluster/home/akmete/Setup/.venv/lib/python3.11/site-packages/joblib/parallel.py", line 763, in _return_or_raise
    raise self._result
joblib.externals.loky.process_executor.TerminatedWorkerError: A worker process managed by the executor was unexpectedly terminated. This could be caused by a segmentation fault while calling the function or by an excessive memory usage causing the Operating System to kill the worker.

The exit codes of the workers are {SIGKILL(-9)}
slurmstepd: error: Detected 1 oom_kill event in StepId=20866524.batch. Some of the step tasks have been OOM Killed.
